builtin/files/userdata/cloud-config-controller

```diff
@@ -2175,20 +5264,35 @@ write_files:
                     value: "first-found"
                   - name: IP6_AUTODETECTION_METHOD
                     value: "first-found"
-                  # Enable IPIP
+                  # Enable or Disable VXLAN on the default IP pool.
                   {{- if .Kubernetes.Networking.SelfHosting.CalicoConfig.VxlanMode }}
                   - name: CALICO_IPV4POOL_VXLAN
                     value: "Always"
                   {{- else }}
+                  # Enable IPIP
                   - name: CALICO_IPV4POOL_IPIP
                     value: "Always"
+                  - name: CALICO_IPV4POOL_VXLAN
```

Comment:
This is potential breaking, and is already configurable, see maybe four lines above.

https://github.com/kube-aws/kube-aws/pull/6/files#diff-bbc025525e7a70447fbc57850d6b8d210d947b8d5ce50b96ccf651aa138e2e13R5268

Reply:
I believe it still is?